### ISSUE TITLE
Migrating to devture-prefixed variables disables postgres-backup?

### ISSUE THREAD

**Issue by cxr6548** [spantaleev/matrix-docker-ansible-deploy#2305]
**Describe the bug**
The first time I ran `git pull && make roles && ansible-playbook -i inventory/hosts setup.yml --tags=setup-all,ensure-matrix-users-created,start -K` after the Nov 28th update I got a message saying that I have data in `/matrix/postgres-backup` but the backup service isn't being used and I can freely delete the data. I foolishly closed the terminal and re-running the setup doesn't produce the message again.

**To Reproduce**
My `vars.yml` file looks like this:

```yaml
matrix_domain: mycomain.com
matrix_homeserver_implementation: synapse

matrix_homeserver_generic_secret_key: 'no'

matrix_ssl_lets_encrypt_support_email: 'myemail'

devture_postgres_connection_password: 'no'

matrix_nginx_proxy_base_domain_serving_enabled: true
matrix_synapse_max_upload_size_mb: 50
matrix_client_element_enabled: false
matrix_synapse_presence_enabled: false

devture_postgres_backup_enabled: true
```

The final line above is what produces the behavior.

**Expected behavior**
I expect to backups to be enabled with new devture prefixes.

**Matrix Server:**
 - OS: Ubuntu 22.04
 - Architecture amd64

**Additional context**

**Comment by spantaleev** [comment]
You shouldn't have renamed `matrix_postgres_backup_*` variables. Only `matrix_postgres_*` variables had to be renamed (except for `matrix_postgres_backup_*`).

This issue brings up a few good points though:

- the changelog entry was confusing and not making it clear that `matrix_postgres_backup` variables were unaffected.. The Ansible task that reported the variables to be renamed [was much better at explaining this](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/d37adfba4e546e4bda3b93d6ccfddf30d54b27b4/roles/custom/matrix_playbook_migration/tasks/validate_config.yml#L38-L48)
- it makes sense for the `matrix-postgres-backup` role to be extracted out just the same, which I've done now - https://github.com/devture/com.devture.ansible.role.postgres_backup . The Matrix playbook has been updated to use this external role. You'll need to rename `matrix_postgres_backup` variables to `devture_postgres_backup` just the same.

In short, from now on, all Postgres-related roles are external and all `matrix_postgres`-prefixed variables can be renamed to `devture_postgres`.
